composer.json

```diff
@@ -16,7 +16,8 @@
     "minimum-stability": "dev",
     "require": {
         "php": ">=7.4",
-        "commongateway/corebundle": "^1.0.86"
+        "commongateway/corebundle": "^1.0.86",
+        "common-gateway/zgw-bundle": "^1.0"
```

Comment:
We should probably require the latest version zgw-bundle 1.0.18 ?

Reply:
I'd say that as long as nothing breaks, we should stick to major versions for required bundles. Only if the bundle is reliant on a function that is introduced in a certain version of the bundle we should require that specific version as base version.